app/js/ui/RoundedIcon/index.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../../css/variable-overrides.scss";
+
+.rounded-icon {
+  padding: 10px;
+}
+
+.rounded-icon__grey {
+  background-color: #939BA1;
```

Comment:
If the color is not in the variables, you can add it. We will probably reuse it.

Reply:
updated, it was but I missed it!!